doc/jp/components/grids/_shared/cascading-combos.md

```diff
@@ -12,7 +12,7 @@ namespace: Infragistics.Controls
 {ComponentTitle} の編集機能では、カスケード コンボを使用する機会が提供されます。前の `Combo` で値を選択すると、ユーザーは次の Combo での選択に関連するデータのみを受け取ります。
 
 ## カスケード コンボを使用した Angular {ComponentTitle} サンプルの概要
-以下のサンプルは、`Grid` がネストされた Cascading `Combo` コンポーネントとどのように動作するかを示しています。
```

Comment:
The localization team asked us to not make any changes in the JP docs because it becomes hard for them to track the changes. So it is probably a good idea to remove any content changed in the JP docs